front-end/src/Routes/HomeRoute.js

```diff
@@ -1,19 +1,19 @@
 import React from "react";
-import Header from "../utils/Header";
+import { Redirect } from "react-router";
 import NumberContainer from "../Numbers/NumberContainer";
+import NotifyHeader from "../utils/NotifyHeader";
 import NumberListFetcher from "../Numbers/NumberListFetcher";
 import { AppContainer } from "../utils/styles";
 
-export default () => (
-  <AppContainer>
+export default ({ location }) => (
+  <AppContainer emptySize="70px">
+    {location.search.includes("tv") && (
```

Comment:
Maybe not necessary to check location.search on every render :thinking:

Reply:
would it be better to make the route a component and cache the result of the includes call?